rules/rules.json

```diff
@@ -27,6 +27,10 @@
         {
             "pattern": ".*player.vimeo.com/(video/[\\d]+)\\?.*",
             "replace": "vimeo.fuzzy.replayweb.page/\\1"
+        },
+        {
+            "pattern": ".*i\\.vimeocdn\\.com\\/(.*)\\?.*",
+            "replace": "i.vimeocdn.fuzzy.replayweb.page/\\1"
```

Comment:
Maybe we could send those rules update upstream ?

Reply:
Definitely, I just wanted to have your feedback first.